pkg/kev/extract.go

```diff
@@ -29,9 +29,9 @@ import (
 
 // extractLabels extracts a set of labels from a compose project object.
 func extractLabels(s composego.ServiceConfig, target *ServiceConfig) {
-	extractServiceTypeLabels(s, target)
-	extractDeploymentLabels(s, target)
-	extractHealthcheckLabels(s, target)
+	// extractServiceTypeLabels(s, target)
```

Comment:
Should these lines be commented out?

Reply:
Yep, I'm progressively removing. these, I'll clean this. up later when they're ready to go

Reply:
OK, maybe just add a TODO comment as it'll be easier to track this later.

Reply:
the function is literally called extractLabels, it will be caught 🤣